src/main/java/org/jabref/gui/entryeditor/LatexCitationsTab.java

```diff
@@ -0,0 +1,139 @@
+import org.jabref.model.texparser.Citation;
+import org.jabref.preferences.PreferencesService;
+
+import org.fxmisc.easybind.EasyBind;
+
+public class LatexCitationsTab extends EntryEditorTab {
+
+    private final LatexCitationsTabViewModel viewModel;
+    private final StackPane searchPane;
+    private final ProgressIndicator progressIndicator;
+    private final ObservableList<VBox> graphicCitationList;
+
+    public LatexCitationsTab(BibDatabaseContext databaseContext, PreferencesService preferencesService,
+                             TaskExecutor taskExecutor) {
+        this.viewModel = new LatexCitationsTabViewModel(databaseContext, preferencesService, taskExecutor);
+        this.searchPane = new StackPane();
+        this.progressIndicator = new ProgressIndicator();
+        this.graphicCitationList = FXCollections.observableArrayList();
+
+        setText(Localization.lang("LaTeX Citations"));
+        setTooltip(new Tooltip(Localization.lang("Search citations for this entry in LaTeX files")));
+        setGraphic(IconTheme.JabRefIcons.APPLICATION_TEXSTUDIO.getGraphicNode());
+    }
+
+    private void setupSearchPane() {
+        progressIndicator.setMaxSize(100.0, 100.0);
+        searchPane.getStyleClass().add("latex-citations-tab");
+
+        setContent(searchPane);
+    }
+
+    @Override
+    protected void bindToEntry(BibEntry entry) {
+        setupSearchPane();
+
+        EasyBind.subscribe(viewModel.statusProperty(), status -> {
+            switch (status) {
+                case IN_PROGRESS:
+                    searchPane.getChildren().setAll(progressIndicator);
+                    break;
+                case CITATIONS_FOUND:
+                    graphicCitationList.setAll(EasyBind.map(viewModel.getCitationList(), this::citationToGraphic));
+                    searchPane.getChildren().setAll(getCitationsPane());
+                    break;
+                case NO_RESULTS:
+                    searchPane.getChildren().setAll(getNotFoundPane());
+                    break;
+                case ERROR:
+                    searchPane.getChildren().setAll(getErrorPane());
+                    break;
+                case INACTIVE:
+                default:
+                    searchPane.getChildren().clear();
+                    break;
+            }
+        });
+
+        viewModel.init(entry);
+    }
+
+    private VBox citationToGraphic(Citation citation) {
+        HBox contextBox = new HBox(new Text(LatexToUnicodeAdapter.format(citation.getContext())));
+        contextBox.getStyleClass().add("latex-citations-context-box");
+
+        Text positionText = new Text(String.format("%n%s (%s:%s-%s)", citation.getPath().toAbsolutePath(),
+                citation.getLine(), citation.getColStart(), citation.getColEnd()));
+        positionText.getStyleClass().add("latex-citations-position-text");
+
+        return new VBox(contextBox, positionText);
+    }
+
+    private ScrollPane getCitationsPane() {
```

Comment:
I guess this method and `citationToGraphic` are (almost) the same as the ones in the dialog. Thus, to reduce code duplication, I would propose to introduce a new ui control that displays a given list of citations.

Reply:
Good. By the way, to show the path, do you prefer `getFileName()` or `toAbsolutePath()`? The first choice may be confusing, but the second one is too long (specially in the dialog). I would like a relative path, from the LaTeX file directory. What do you think about?

Reply:
I think showing the relative path is a good choice.

Reply:
Finally, I have created a new UI wrapper (`CitationViewModel`) that contains a `getDisplayGraphic()` method. Hope that is fine too.